Firestore/core/src/firebase/firestore/local/leveldb_migrations.cc

```diff
@@ -184,6 +189,39 @@ void RemoveAcknowledgedMutations(leveldb::DB* db) {
   transaction.Commit();
 }
 
+/** Migration 6. */
```

Comment:
This seems the right way to make this work.

This is only really useful in that it leaves the data in a state that's compatible with previous versions that did use this field. This would allow developers to rollback safely to prior versions.